packages/ffe-core/bin/build.js

```diff
@@ -0,0 +1,62 @@
+const fs = require('fs');
+const path = require('path');
+const Case = require('case');
+const mkdirp = require('mkdirp');
+
+// These are the files we want to convert to JavaScript
+const FILES_WITH_VARIABLES = ['colors', 'breakpoints'];
+
+// First, create the lib directory if it doesn't exist
+mkdirp.sync(path.resolve(__dirname, '..', 'lib'));
+
+// Then, let's process each file!
+FILES_WITH_VARIABLES.forEach(filename => {
+    const lessContent = fs.readFileSync(
+        path.resolve(__dirname, '..', 'less', `${filename}.less`),
+        'utf8'
+    );
+
+    // This is done in a ridiculously step by step fashion to increase
+    // readability. Basically, this changes a LESS file into a JavaScript object
+    const variables = lessContent.split('\n')
+        .map(line => line.trim())
+        .filter(line => line.startsWith('@'))
+        .map(line => line.replace('@ffe-', ''))
+        .map(line => line.replace('@', ''))
+        .map(line => line.split(': '))
+        .reduce((allVars, [key, value]) => ({
+            ...allVars,
+            [Case.camel(key)]: value.substring(0, value.indexOf(';')),
+        }), {});
+
+    // Create the output string, complete with a header explaining where it came
+    // from and how to change it.
+    const jsContent = `// DO NOT MODIFY!
+// This file is created automatically by ffe-core/bin/build.js, and is based on
+// the ffe-core/less/${filename}.less file. Change the value there instead.
+
+module.exports = ${JSON.stringify(variables, null, 4)};
+`;
+
+    // Write the formatted string to its own file
+    fs.writeFileSync(
+        path.resolve(__dirname, '..', 'lib', `${filename}.js`),
+        jsContent,
+    );
+});
+
+// Finally, let's create an index file that exports all variables
```

Comment:
I'm a bit torn here. It would be safer (with regards to name collisions) to not have a shared index and import from `ffe-icons/colors` like we did in older versions of `icons-react`.

On the other hand, chances are low that we'll ever get a naming collision and I know we just refactored `ffe-icons-react` to use a single shared `index.js`. On the other _other_ hand, there we have a single folder and unique file names that get translated to exports. Here we have no such guarantee.

Thoughts?

Reply:
> LESS-variables are global

Good point! Then it's a non-issue. Carry on :)